src/Evolution/Systems/Ccz4/Christoffel.cpp

```diff
@@ -0,0 +1,65 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cstddef>
+
+#include "Evolution/Systems/Ccz4/Christoffel.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/IndexType.hpp"
```

Comment:
don't need `IndexType.hpp` and `TypeAliases.hpp`, these are considered as exposed by `Tensor.hpp`. I don't remember, do you need to add an include for the expressions?

Reply:
No, I it's the same as what you're describing regarding `IndexType.hpp` and `TypeAliases.hpp`, since `Tensor.hpp` includes all TE files